developer/ruby/ApplyCopyright.rb

```diff
@@ -1,34 +1,34 @@
-# Inserts copyright at beginning of each
-#
-# Inputs:
-#   ARGV[0] - path to top level cmake source directory (one level above 'src' directory)
+# Inserts copyright at beginning of each ruby/
 
 require 'pathname'
 require 'rubygems'
 require 'fileutils'
 
 include FileUtils
 
-# check that called from command line directly
-if not ($0 == __FILE__)
-  puts "#{__FILE__} called from external script"
-  exit
-end
 
-basepath = ARGV[0].gsub("\\", "/")
+ROOT_DIR = File.absolute_path(File.join(File.dirname(__FILE__), "../../"))
+LICENSE_PATH = File.join(ROOT_DIR, 'LICENSE.md')
+
+
+license_lines = File.readlines(LICENSE_PATH)
+# Now with short version
+license_lines = [
+  "OpenStudio(R), Copyright (c) Alliance for Sustainable Energy, LLC.\n",
+  "See also https://openstudio.net/license\n"
+]
```

Comment:
New short license here. Shouldn't change.

the gttps://openstudio.net/license is a dead link currently though